src/realm/cluster_tree.cpp

```diff
@@ -914,13 +988,6 @@ bool ClusterTree::is_valid(ObjKey k) const noexcept
     return m_root->try_get(k, state);
 }
 
-ClusterNode::State ClusterTree::get(ObjKey k) const
```

Comment:
I don't understand this either, how do we fetch an Obj? Only via insertion?

Reply:
I think you got the answer yourself further down,